samples/contrib/pytorch-samples/cifar10/cifar10_train.py

```diff
@@ -205,7 +206,3 @@ def training_epoch_end(self, outputs):
         """
         self.show_activations(self.reference_image)
 
-        # Logging graph
-        if self.current_epoch == 0:
-            sample_img = torch.rand((1, 3, 64, 64))
-            self.logger.experiment.add_graph(CIFAR10Classifier(), sample_img)
```

Comment:
Is this not needed anymore?

Reply:
Yes @chauhang This is handled by tensorboard - https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.loggers.tensorboard.html